source/includes/steps-starting-compass.yaml

```diff
@@ -60,9 +67,12 @@ content: |
    the :guilabel:`Connect` button to navigate to the |compass-short|
    :doc:`Home Page </instance>`.
 
-   The home page |home| provides details regarding
-   the MongoDB instance to which |compass-short| is connected. It shows
-   the hostname and port, the deployment's MongoDB version, and a list
-   of the deployment's :doc:`databases </databases>`. For more
-   information on this page, see :doc:`/instance`.
+   .. admonition:: Required Access
```

Comment:
I feel like this should be Yellow or Red, but not Green. A lot of people complain that they can't see certain tabs in Compass after logging in. We should highlight this note to do our best to help them out.